server/src/api/api.graphql

```diff
@@ -78,3 +84,54 @@ type Category {
     # The human-readable, user-facing name for this category
     category_name: String!
 }
+
+enum RequestStatus {
+	SUBMITTED
+	APPROVED
+	DENIED
+	ABANDONED
+    CANCELLED
+	READY_FOR_PICKUP
+	FULFILLED
+	RETURNED
+	LOST
+	DAMAGED
+}
+
+type Request {
+	request_id: Int!,
+	user: User!,
```

Comment:
don't you only need `user_id` and `item_id`? Is the merging scheme like so:
requests in pending / pre-return status are uniquely identifiable by status-user-item OR request id. After return status, they're only identifiable by request id.

basically, after return only status changes can come in. Changes pre-return will clump together in the backend?

Reply:
Even if you could identify by that compound key it's nice to have a request id for quick identification